### ISSUE TITLE
Broken on Safari 9

### ISSUE THREAD

**Issue by bergie** [noflo/noflo-ui#521]
Both Safari 9.0.1 on OS X and Mobile Safari:

<img width="1242" alt="screen shot 2015-12-01 at 20 12 24" src="https://cloud.githubusercontent.com/assets/3346/11512824/fc3b897a-9867-11e5-8830-672a0824d215.png">

Sounds like a time to revive the Polymer 1.0 conversation the-grid/the-graph#222

We're currently in Polymer 0.5.2. Same issue reported in dart-lang/polymer-dart#538 with mention that upgrading the `web_components` polyfill might fix it.

**Comment by bergie** [comment]
Upstream issue in webcomponents/webcomponentsjs#348

**Comment by rraallvv** [comment]
Is this still an issue on Safari?
I tried to install noflo-ui following the instruction in the README file but it threw the error `Cannot find module coffee-script`. I installed the dependency with the command `npm install --save-dev coffee-script`. Also needed to install `coffeelint` because of another error. I didn't start the watcher process. Then I launched the server and tried to load the noflo from safari but the there are some issues:
- sometimes when I try to open an example it won't load
- when it load it doesn't show see the expected output, if I click the button to run the example it opens the preview sidebar but all it does it to load a black rectangle occupying all the preview area.

![screen shot 2016-04-10 at 9 12 14 pm](https://cloud.githubusercontent.com/assets/1840524/14414765/030c07e2-ff61-11e5-88a4-13f00f2b3938.png)
